applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/app_ZDebug.conf

```diff
@@ -135,8 +135,11 @@ CONFIG_RESET_ON_FATAL_ERROR=n
 
 CONFIG_LOG=y
 CONFIG_LOG_PRINTK=y
+CONFIG_LOG_BUFFER_SIZE=4096
+CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=1024
+CONFIG_LOG_PROCESS_THREAD_STARTUP_DELAY_MS=10000
```

Comment:
Does it have to be that big value to solve the problem you mention?  I think 10 seconds is big enough that user will surely spot it and wonder why is it happening. I believe that something like 3-5 seconds would result in giving MCU time to initialize necessary modules and it would be a value small enough that before user will manage open serial port, the log should appear just afterwards.

Reply:
Ok changed to CONFIG_LOG_PROCESS_THREAD_STARTUP_DELAY_MS=4000.